lib/dotcom_web/components/trip_planner/leg_to_segment_helper.ex

```diff
@@ -0,0 +1,37 @@
+defmodule DotcomWeb.Components.TripPlanner.LegToSegmentHelper do
```

Comment:
I'm not sold on either this name or this location for this file, but I couldn't think of a better one off the top of my head.

Reply:
Since it's not technically a rendered component, you can move this to the `Dotcom.TripPlan.` namespace. I don't mind the current name.

Reply:
💥 https://github.com/mbta/dotcom/pull/2262/commits/5caf9b799796fbd1980a5ae5b5aa4b4d11c5b1de 💥